modules/osu_groups_basic_group/tests/src/Kernel/OsuGroupsBasicGroupPathKernelTest.php

```diff
@@ -0,0 +1,200 @@
+    $this->installEntitySchema('group_content_menu');
+    $this->installEntitySchema('menu_link_content');
+    if ($this->container->get('entity_type.manager')
+      ->hasDefinition('path_alias')) {
+      $this->installEntitySchema('path_alias');
+    }
+    $this->installConfig(['pathauto', 'system', 'node', 'group']);
+    $this->installSchema('node', ['node_access']);
+
+    // Create a node type.
+    $type = NodeType::create(['type' => 'page']);
+    $type->save();
+    node_add_body_field($type);
+
+    // Create a group type.
+    $group_type = GroupType::create([
+      'id' => 'basic_group',
+      'label' => 'Basic',
+      'new_revision' => TRUE,
+      'creator_membership' => TRUE,
+      'creator_wizard' => FALSE,
+    ]);
+    $group_type->save();
+
+    // Create a menu.
+    $menu = Menu::create(['id' => 'main_menu', 'title' => 'Main Menu']);
+    $menu->save();
+
+    // Create group content menu type.
+    $group_menu_type = GroupContentMenuType::create([
+      'id' => 'group_menu',
+      'label' => 'Group Menu',
+    ]);
+    $group_menu_type->save();
+    /** @var \Drupal\group\Entity\Storage\GroupContentTypeStorageInterface $storage */
+    $storage = $this->entityTypeManager->getStorage('group_content_type');
+    // create group content from plugin
+    $storage->createFromPlugin($group_type, 'group_content_menu:group_menu')
+      ->save();
+    $storage->createFromPlugin($group_type, 'group_node:page', [
+      'group_cardinality' => 1,
+      'entity_cardinality' => 1,
+      'use_creation_wizard' => FALSE,
+    ])->save();
+
+    // Set config for pathauto to create new aliases and delete old ones (Default).
+    $config = $this->config('pathauto.settings');
+    $config->set('update_action', PathautoGeneratorInterface::UPDATE_ACTION_DELETE);
+    $config->save();
+
+    // Add pathauto patterns.
+    $this->nodePattern = $this->createPattern('node', '[node:menu-link:parent:url:path]/[node:title]');
+    $this->nodePattern->save();
+    $this->groupPattern = $this->createPattern('group', '[group:title]');
+    $this->groupPattern->save();
+    \Drupal::service('router.builder')->rebuild();
+
+    // Create a new user.
+    $this->createUser();
+    $this->setCurrentUser($this->createUser());
+  }
+
+  public function testOsuGroupsAlias() {
+    // Create first node.
+    $node_1 = Node::create([
+      'title' => 'About Us',
+      'type' => 'page',
+    ]);
+    $node_1->setPublished();
+    $node_1->save();
+
+    // Put it in the menu.
+    $link_1 = MenuLinkContent::create([
+      'title' => 'About Us',
+      'link' => ['uri' => 'entity:/node/' . $node_1->id()],
+      'menu_name' => 'main_menu',
+      'weight' => 0,
+    ]);
+    $link_1->save();
+
+    // Create a second node that will be in the group.
+    $node_2 = Node::create([
+      'title' => 'About Us',
+      'type' => 'page',
+    ]);
+    $node_2->setPublished();
+    $node_2->save();
+
+    // Crate a group.
```

Comment:
Create

Reply:
thanks for the catch. Pushed a fix for the spelling error.